### DISCUSSION TITLE
34. SOP의 개념과 함께 CORS의 등장 배경을 설명해주세요.

### DISCUSSION THREAD

**Discussion by linirini** [jewelry-box/cs-interview#35] [General]
.

**Comment by jongmee** [comment]
SOP(Same Origin Policy)는 같은 출처의 자원만 브라우저가 사용하게 하는 보안 정책이다. CSRF와 같은 공격을 예방하고 보안을 강화하고자 존재한다. 이 환경에서 CORS는 다른 출처의 자원도 브라우저가 사용할 수 있게 해주는 매커니즘이다.
